lua/obsidian/util.lua

```diff
@@ -316,4 +317,30 @@ util.working_day_before = function(time)
   end
 end
 
+---Get the filename of the note under the cursor.
+---
+---@return string note name
+---@return string note file name
+util.get_note_name = function()
```

Comment:
I think there's an issue with this function. If your cursor is on a link that looks like this: `[[NOTE_ID|Note title]]`, the `note_name` returned will be "NOTE_ID|Note title", when it should be "Note title".